File: examples/employee-directory-laravel/database/migrations/2026_08_22_000002_create_billing_core_links.php
<?php

use Illuminate\Database\Migrations\Migration;
use Illuminate\Database\Schema\Blueprint;
use Illuminate\Support\Facades\Schema;

return new class extends Migration
{
    public function up(): void
    {
        Schema::create('billing_core_links', function (Blueprint $table) {
            $table->id();
            $table->foreignId('organization_id')->unique()->constrained()->cascadeOnDelete();
            $table->string('customer_ref');
            $table->string('contract_ref');
            $table->string('subscription_ref');
            $table->string('plan_version_ref');
            $table->timestamps();
        });
    }

    public function down(): void
    {
        Schema::dropIfExists('billing_core_links');
    }
};

File: examples/crm-rails/db/migrate/20260822000002_create_billing_core_links.rb
class CreateBillingCoreLinks < ActiveRecord::Migration[8.1]
  def change
    create_table :billing_core_links do |t|
      t.references :organization, null: false, foreign_key: true, index: { unique: true }
      t.string :customer_ref, null: false
      t.string :contract_ref, null: false
      t.string :subscription_ref, null: false
      t.string :plan_version_ref, null: false
      t.timestamps
    end
  end
end
